src/common/latest/sonarqube/Scanner.ts

```diff
@@ -263,7 +286,12 @@ export class ScannerMSBuild extends Scanner {
     if (msBuildVersion) {
       // Download the specified scanner version
       const downloadUrl = scannerConfig.msBuildUrlTemplate(msBuildVersion, useNetFramework);
-      const scannerArchivePath = await Scanner.downloadScanner(downloadUrl);
+      const scannerArchivePath = await Scanner.downloadScanner(
+        downloadUrl,
+        "SonarScanner MSBuild",
```

Comment:
Could we use the new `.NET` name here already?

```suggestion
        "SonarScanner .NET",
```